docs/_advanced/error_handling.md

```diff
@@ -17,3 +17,37 @@ app.error((error) => {
   console.error(error);
 });
 ```
+
+<details class="secondary-wrapper">
+<summary class="section-head" markdown="0">
+<h4 class="section-head">Accessing more data in the error handler</h4>
+</summary>
+
+<div class="secondary-content" markdown="0">
+There may be cases where you need to log additional data from a request in the global error handler. Or you may simply wish to have access to the `logger` you've passed into Bolt. 
+
+When passing `extendedErrorHandler: true` to the constructor, the error handler will receive an object with `error`, `logger`, `context`, and the `body` of the request.    
```

Comment:
How about mentioning the supported version? For example, "To use this option, you will need Bolt for JavaScript version 3.8.0 or newer." or something like that.


https://slack.dev/bolt-js/reference